### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by dubstard**
add 
        "globalsyncwallet.top",
image
haha, open dir
image

        "dappsmigration.online",
image

        "dappsmigration.xyz",
image
         
        "dexlaunchpads.org",
image

        "onlineearnconnect.xyz",

### CODE REVIEW

**Review by jacogr — DISMISSED** [review]

**Review by dubstard — COMMENTED** [review]
fix fat finger mess

File: all.json [hunk 1]
```diff
@@ -5617,7 +5623,6 @@
 		"validatewallets.net",
 		"validatewallets.webflow.io",
 		"validatewalletserror.com",
-		"validateweb.org",
```
Comment (jacogr) [inline all.json:5620]: Why is this removed? Invalid?

Comment (jacogr) [reply all.json:5620]: ... Chrome does think it is suspect.

Comment (dubstard) [reply all.json:5620]: Nope, copy pasta error ( i was checking if we already had it, and instead of deleting the dupkicate removed the real one, re adding) 
Sorry I shouldn't had messed with it after the review and should had opened a new one instead :/

**Review by dubstard — COMMENTED** [review]
Re-added "validateweb.org",  🤦‍♂️

**Comment by dubstard** [comment]
Sorry. Should be good now.🤞

**Review by jacogr — APPROVED** [review]

**Comment by dubstard** [comment]
apologies for the mess, fixed
